scripts/chroot.sh

```diff
@@ -155,10 +155,9 @@ apt-get install -f
 apt-get -qq -y remove dconf-tools
 
 # Instalation of packages from Generator Webapp
-SCRIPT_URL=https://www.github.com/fossasia/meilix-generator/archive/master.zip
-wget -O $scripts.zip $SCRIPT_URL
-unzip scripts.zip
-SCRIPTS_FOLDER_IN_ZIP="meilix-generator-master/scripts"
+SCRIPT_URL=https://github.com/fossasia/meilix-generator/trunk/scripts
```

Comment:
ok it will work just in case multiple users are not using the generator at once

Reply:
Yes, it seems similar to me too